webview/src/setup/components/App.tsx

```diff
@@ -50,13 +50,8 @@ const getDvcStatusIcon = (
     : TooltipIconType.PASSED
 }
 
-const getStudioStatusIcon = (cliCompatible: boolean, isConnected: boolean) => {
-  if (!cliCompatible) {
-    return TooltipIconType.ERROR
-  }
-
-  return isConnected ? TooltipIconType.PASSED : TooltipIconType.INFO
-}
+const getStudioStatusIcon = (cliCompatible: boolean, isConnected: boolean) =>
+  cliCompatible && isConnected ? TooltipIconType.PASSED : TooltipIconType.ERROR
```

Comment:
[C] I guess this could be `TooltipIconType.WARNING` instead but up to you.

Reply:
I forgot we even had that 🤦🏻

Reply:
Oh yeah, warning makes sense! Will also help with users knowing that its optional.